CONTRIBUTING.md

```diff
@@ -34,15 +34,11 @@ If you don't have npm, it comes with node. To get it via homebrew:
 `brew install node`
 
 To add or update a dependency:
-- `go get -u` to update the dependencies or `go get {package}` to add a dependency
+- `cat GLOCKFILE | grep -oE '[^[:space:]]+\.[^[:space:]]+' | sort | uniq | xargs go get -u` to update the dependencies or `go get {package}` to add a dependency
```

Comment:
`glock` is inherently problematic for people who work on other projects in the same `GOPATH`. We should just assume that we're the only thing on the `GOPATH` (and document things accordingly), at least for people who feel enough ownership of the product that they'd try a blind update of all dependencies simultaneously.